README.md

```diff
@@ -253,7 +253,52 @@ but, if you want to (change the system based on your system):
      };
    }
    ```
-7. NixOS rebuild or use your deployment tool like usual.
+7. (Optional) Setup git to decrypt `*.age` files when showing diffs
```

Comment:
I don't think we should insert optional steps in the tutorial. It is hard enough to follow as is. I'd be okay with adding this as a separate section though!